src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -77,11 +77,15 @@ export type ModelingCommandSchema = {
     thickness: KclCommandValue
   }
   Revolve: {
+    // Enables editing workflow
+    nodeToEdit?: PathToNode
+    // Flow arg
+    axisOrEdge: 'Axis' | 'Edge'
+    // KCL stdlib arguments
     selection: Selections
     angle: KclCommandValue
-    axisOrEdge: 'Axis' | 'Edge'
-    axis: string
-    edge: Selections
+    axis: string | undefined
+    edge: Selections | undefined
```

Comment:
Is this `undefined` because we prompt them with the same `Revolve` command but since we only let them edit the `angle` these other inputs could be `undefined` in that same workflow?

Reply:
Yes! Not 100% because on edit we actually massage the artifact into an original selection so that actually exist, but when you pick axisOrEdge one of those is going to remain undefined so this typing is more true. Like you never get an axis string _and_ an edge selection.